pyclam/manifold.py

```diff
@@ -115,6 +112,13 @@ def parent(self) -> 'Cluster':
                 raise ValueError(f"root cluster has no parent")
         return self.cache['parent']
 
+    @property
+    def color(self) -> str:
+        """ The color of the cluster node, derived from its LFD """
+        x = 29 + ((19 * 2) * int(round(self.local_fractal_dimension)))
+        y = 206 - ((22 * 2) * int(round(self.local_fractal_dimension)))
+        return '#' + '%02x%02x%02x' % (x, 0, y)
```

Comment:
I would prefer to see f-strings.

Reply:
f-strings are our preferred default for string formatting. They are much easier to read, and are faster in execution.